source/docs/articles/sites/code/unsupported-modules-plugins.md

```diff
@@ -6,56 +6,92 @@ keywords: modules, plugins, unsupported, drupal, wordpress
+<hr>
+### Apache Solr Search
+**Issue**: If Acquia Solr modules are present in the site codebase (even if disabled) and Pantheon Apache Solr is enabled, the site will be unable to connect to Solr server.
+
+**Solution**: Delete the Acquia Solr modules from the codebase and then disable and re-enable the Pantheon Apache Solr module.
+<hr>
+### Background Process  
+**Issue**: The module allows for Drupal to perform "parallel" (asynchronous non-blocking mode) requests. However, there are a number of limitations working in a distributed environment and function correctly on the platform.
 
-- Adaptive Image Styles
+<hr>
+### Backup & Migrate
+**Issue**: The Backup and Migrate module can create large archives and cause issues with the workflow tools on the dashboard. See [Backup Creation](/docs/articles/sites/backups/backup-creation/#drupal-note-why-is-backup-&-migrate-not-recommended-on-pantheon?).
 
-- Apache Solr Search
-**Issue** If Acquia Solr modules are present in the site codebase (even if disabled) and Pantheon Apache Solr is enabled, the site will be unable to connect to Solr server.  
-**Solution** Delete the Acquia Solr modules from the codebase and then disable and re-enable the Pantheon Apache Solr module.
+**Solution**: You can use the automated backups that are available on the dashboard for each environment.
+<hr>
+### Boost
+**Issue**: We do not recommend the boost module on the platform as Varnish is running. Running Boost in addition to Varnish can cause caching issues.
 
-- Background Process  
-**Issue**: The module allows for Drupal to perform "parallel" (asynchronous non-blocking mode) requests. However, there are a number of limitations working in a distributed environment and function correctly on the platform.
+<hr>
 
-- Backup & Migrate
+### Cache Expiration
+**Issue**: Unfortunately, there is no way to selectively purge the Varnish cache.
 
-- Boost
+**Solution**: You can set a `NO_CACHE` cookie or set `Cache-Control: no-cache,max-age=0` header to bypass Varnish.
 
-- Cache Expiration
 
-- Global Redirect  
+<hr>
+### Global Redirect  
  **Issue**: Too many redirects error when site is in maintenance mode.  
- **Solution**: Ensure that the "Frontpage Redirect Handler" is not ticked in the Global Redirect administration page.
 
-- HTTPRL  
+ **Solution**: Ensure that the "Frontpage Redirect Handler" is not ticked in the Global Redirect administration page.
+<hr>
+### HTTPRL  
 **Issue**: This module can severely impact performance. This may be the result of module code or its configuration on the platform that results in the spikes.
 
-- IMCE 6.x
+<hr>
+### IMCE 6.x
+**Issue**: Operations on directories containing an inordinate amount of files will likely hit the load balancer timeout threshold (30 seconds).
 
-- Mobile Tools
+**Solution**: The alternative for now is to break up the files into smaller groups.
```

Comment:
@ari-gold found this patch as well: https://www.drupal.org/node/1702976

The first one you linked was created 3 years ago. This one is more recent and seems to be applicable to the issue stated here. Solves performance issues on large directories with pagination. Thoughts? 

I didn't create a new issue since I found similar ones posted in the project.